block-node/block-providers/files.recent/src/main/java/org/hiero/block/node/blocks/files/recent/BlockFileRecentPlugin.java

```diff
@@ -273,6 +283,12 @@ public void handleVerification(VerificationNotification notification) {
                     }
                 }
             }
+            final long endTime = System.nanoTime() - startTime;
```

Comment:
Naming is a little confusing
```suggestion
            final long totalTime = System.nanoTime() - startTime;
```

Reply:
✅ Done!